docs/sources/operator/getting-started.md

```diff
@@ -1,162 +1,151 @@
 ---
 aliases:
 - /docs/agent/latest/operator/getting-started/
-title: Installing Grafana Agent Operator
-weight: 100
+title: Install Grafana Agent Operator without Helm
```

Comment:
I would just leave it as it is and not mention helm in the title. This is the default way to install resources in kubernetes.

Reply:
Sounds good.